src/pages/Patients.jsx

```diff
@@ -1,17 +1,20 @@
-import { useState, useMemo } from "react";
+import { useMemo } from "react";
 import { AgGridReact } from "ag-grid-react";
 import "ag-grid-community/styles/ag-grid.css";
 import "ag-grid-community/styles/ag-theme-alpine.css";
+import { useQuery } from "@tanstack/react-query";
+import { getPatients } from "../services/apiPatients";
 const Patients = () => {
-  const [rowData] = useState([
-    {
-      patientID: "CHCP1045",
-      patientName: "GERALD ARNORLD",
-      patientEmail: "epipat.pe@gmail.com",
-      clinicianAssignedId: "CHCC0008",
-      status: "IN PROGRESS IN PROGRESS",
-    },
-  ]);
+  const {
+    isPending,
+    data: patients,
+    error,
+  } = useQuery({
+    queryKey: ["patients"],
+    queryFn: getPatients,
+  });
+
```

Comment:
empty line . needs to be closed

My understanding is that we are trying to avoid overheads due to empty spaces or any redundant comments

Reply:
Fixed